src/main/java/net/helpscout/api/ApiClient.java

```diff
@@ -79,57 +84,80 @@
 
 public class ApiClient {
```

Comment:
Yeah, staying backward compatible definitely restricted some of the refactoring efforts. Overall I think this PR fixes a lot of headaches customers were experiencing, but I'd like to eventually roll out v 2.0 of this library and ditch backward compatibility in favor of a cleaner API. I've created an internal task to eventually start that effort.